README.md

```diff
@@ -52,7 +52,7 @@ Location.select { |x| x.calendars.any? }.each { |x| x.update(keeps_hours: true)
 
 To run the app locally while logged in as an authorized user for the backend, run the following:
 ```
-REMOTE_USER=username@stanford.edu eduPersonEntitlement=group bin/dev
+REMOTE_USER=username@stanford.edu eduPersonEntitlement=dlss:staff bin/dev
```

Comment:
Because of this line, I actually think `group` is best for dev: `https://github.com/sul-dlss/library_hours_rails/blob/3f1345b3b1d7724dfd8f5297981105f2f588f7e7/config/settings/development.yml#L2

With `dlss:staff` I didn't have the permisison to edit